pilot/helpers/cli.py

```diff
@@ -235,21 +221,36 @@ def execute_command(project, command, timeout=None, success_message=None, comman
                 output += line
                 print(color_green('CLI OUTPUT:') + line, end='')
                 logger.info('CLI OUTPUT: ' + line)
-                if success_message is not None and success_message in line:
-                    logger.info('Success message found: %s', success_message)
-                    # break # TODO background_command - this is if we want to leave command running in background but sometimes processes keep hanging and terminal gets bugged, also if we do that we have to change user messages to make it clear that there is command running in background
-                    raise CommandFinishedEarly()
+                # if success_message is not None and success_message in line:
```

Comment:
Why is this no longer needed?

Reply:
expected succes_message when running command is generated by LLM and he has to "predict" the expected output. Sometimes it gives us message that happens when we start server and we terminate process running command too early because of it. 
 
 This is temporarily removing that feature.